axis.go

```diff
@@ -248,9 +248,9 @@ func (a *horizontalAxis) GlyphBoxes(*Plot) (boxes []GlyphBox) {
 		w := a.Tick.Label.Width(t.Label)
 		box := GlyphBox{
 			X: a.Norm(t.Value),
-			Rectangle: draw.Rectangle{
-				Min: draw.Point{X: -w / 2},
-				Max: draw.Point{X: w / 2},
+			Rectangle: vg.Rectangle{
```

Comment:
it's already part of that PR.
this PR is rebased on top of it, but as #261 hasn't been merged yet, it's shown in the set of CLs.
(perhaps I should have done things differently when creating this PR...)